src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.Windows.cs

```diff
@@ -15,6 +15,8 @@ public class NetworkChange
         //introduced for supporting design-time loading of System.Windows.dll
         [Obsolete("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)]
         public static void RegisterNetworkChange(NetworkChange nc) { }
+        
+        static private object globalLock = new object();
```

Comment:
'private' should come before 'static'

Reply:
And variable name should have "s_" prefix as per CoreFx coding guidelines.

Reply:
Will fix.